packages/fetch-http-handler/src/fetch-http-handler.ts

```diff
@@ -77,7 +77,7 @@ export class FetchHttpHandler implements HttpHandler<FetchHttpHandlerConfig> {
     }
     const requestTimeoutInMs = this.config!.requestTimeout;
     const keepAlive = this.config!.keepAlive === true;
-    const credentials = this.config!.credentials;
+    const credentials = this.config!.credentials as RequestCredentials;
```

Comment:
the point of the change was to remove usage of a global type, this only moves it to here.